### DISCUSSION TITLE
Call for Mentors for Open Source Promotion Plan 2024!

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#5106] [General]
@apolloconfig/committers @apolloconfig/members 

Apollo community has signed up for [Open Source Promotion Plan](https://summer-ospp.ac.cn/), which encourages university students to participate in the development and maintenance of open-source software. Students can apply for the project they are interested in and guided by the senior maintainer (community mentor) once they are selected.

This is a great opportunity to cultivate fresh blood in the open-source community, and I strongly encourage committers and members to sign up for the mentor and contribute tasks for the students to participate.
If you want to be a mentor but don't have a task in your mind, then you may use choose from the following candidates:
* #4697 
* #4327 
* #4243
* #3066

Here is the timeline:

| **Date** | **Event** |
| ----------- | ---------------------- |
| 03/11-03/31 | Community Registration and Review |
| 04/04-04/29 | Project Publish and Review |
| 04/30-06/04 | Student Registration, Mentor Communication, and Project Application |
| 06/05-06/25 | Project Application Review |
| 06/26 | Accepted Student Projects Announced |
| 06/26-06/30 | Community Bonding |
| 07/01-09/30 | Coding and Development |
| 10/01-10/31 | Evaluations – PR/MR Merge and Mentor Final Term Review |
| 11/01-11/08 | Evaluations – Committee Final Term Review |
| 11/09 | Results Announced |
| November | Outstanding Students Selection |

**Comment by hezhangjian** [comment]
I want to be a mentor. And I also have several interesting topics:
- Client cache support kubernetes configmap, allow share cache across deployment or kubernetes
- Support RateLimit control in ConfigServer, there may be many client connects to configservice
- Support other database like GaussDB

**Comment by klboke** [comment]
+1

- https://github.com/apolloconfig/apollo/issues/5107

**Comment by Anilople** [comment]
+1

* #5113

**Comment by nobodyiam** [comment]
@shoothzj @klboke @Anilople 

I've created a [wiki page](https://github.com/apolloconfig/apollo/wiki/Open-Source-Promotion-Plan-Summer-2024-Tasks) dedicated to our tasks. Could you assist in populating it with the necessary details? This will enable us to submit our projects to the [Open Source Promotion Plan](https://summer-ospp.ac.cn/).
